src/main.cpp

```diff
@@ -203,16 +228,73 @@ void rpcWSWrapper() {
 void GetDeviceInfo() {
   JsonDocument jsonResponse;
   jsonResponse["name"] = shelly_name;
-  jsonResponse["id"] = shelly_name;
+  jsonResponse["id"] = shelly_name; // ???? ToDo check

```

Comment:
ist das Verhalten von uni-meter...

Reply:
id = shelly_name ist laut API korrekt, allerdings gibt es eigentlich name nicht in der Antwort auf Shelly.GetDeviceInfo...

Reply:
Wenn es in der API drin steht, so würde ich es drin lassen. Es kann durchaus möglich sein, das es durch andere Komponenten abgefragt wird. Bsp: FHEM (da müsste ich den Test wagen ob es raus kann) aber ich sehe Uni-Meter auch nicht als "orginal" da kann es sein, das dort vielleicht auch die Auswertung nicht übernommen wurde.